fixed-hash/src/hash.rs

```diff
@@ -418,24 +500,28 @@ macro_rules! impl_heapsize_for_hash {
 				0
 			}
 		}
-	}
+	};
 }
 
-#[cfg(any(not(feature="heapsizeof"), not(feature="libc"), target_os = "unknown"))]
+#[cfg(any(
+	not(feature = "heapsizeof"),
+	not(feature = "libc"),
+	target_os = "unknown"
+))]
 #[macro_export]
 #[doc(hidden)]
 macro_rules! impl_heapsize_for_hash {
-	($name: ident) => {}
+	($name: ident) => {};
 }
 
-#[cfg(feature="std")]
+#[cfg(feature = "std")]
 #[macro_export]
 #[doc(hidden)]
 macro_rules! impl_std_for_hash {
 	($from: ident, $size: tt) => {
 		impl $from {
 			/// Get a hex representation.
-			#[deprecated(note="Use LowerHex or Debug formatting instead.")]
+			#[deprecated(note = "Use LowerHex or Debug formatting instead.")]
```

Comment:
This has been deprecated for a long time, let's remove it.

Reply:
good idea! :)